plugins/README.md

```diff
@@ -1,18 +1,14 @@
 # Nextflow plugins
```

Comment:
Any reason for keeping this file here at all? Could we not have a section of the main docs called something like "Developing plugins"?

Reply:
I will try to add this as a page on "core plugins"